views/DashboardContainer.tsx

```diff
@@ -16,58 +18,62 @@ class DashboardContainer extends React.PureComponent<DashboardContainerProps> {
   render() {
     const { gps, batteries, windSensors } = this.props;
 
-    const gpsChartData = gps.data.map((gpsPoint) => ({
-      timestamp: gpsPoint.timestamp,
-      speed: gpsPoint.speed,
-    }));
+    const gpsChartData = [
+      this.props.gps.data.map(data => this._parseISOString(data.timestamp)),
+      this.props.gps.data.map(data => data.speed),
```

Comment:
Just to keep this consistent with the rest of your implementation, let's use `gps.data.map` and remove the `this.props` in front of it.